infra/README.md

```
@@ -0,0 +1,237 @@
+# Azure Infrastructure - Naming Conventions
+
+This infrastructure follows the **NeuralLiquid Azure Naming Standards v3**.
+
+## Naming Pattern
+
+All resources follow the pattern:
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)